src/java/org/apache/cassandra/config/YamlConfigurationLoader.java

```diff
@@ -519,6 +559,11 @@ private static void addReplacement(Class<?> klass,
             // by default deprecated is false
             this.deprecated = deprecated;
         }
+
+        public boolean isValueFormatReplacement()
```

Comment:
this got moved to its own top-level class, so will be a merge conflict on rebase